hack/olm-registry/olm-artifacts-template.yaml

```
@@ -28,6 +28,8 @@ parameters:
   required: true
 - name: AWS_MANAGED_TAGS
   required: false
+- name: FEDRAMP
+  required: false
```

We should default the value to "false" to make sure we don't break commercial app-interface/OSD.
```suggestion
  required: false
  value: "false"
```